data/campaigns/Heir_To_The_Throne/scenarios/17_Scepter_of_Fire.cfg

```diff
@@ -3,408 +3,22 @@
     id=17_Sceptre_of_Fire
     name= _ "The Sceptre of Fire"
     next_scenario=18_A_Choice_Must_Be_Made
-    scenario_generation=cave
-
-    #
-    # If you think this is laying it on a bit thick, I'm open to
-    # suggestions for better wording or something less
-    # melodramatic.
-    #
-    [story]
-        [part]
-            story= _ "Across the chasm, the air almost crackled with magical energy. It also became steadily warmer. The floor was smooth and glassy in places, and a faint glow provided a small reprieve from the thick blackness. Volcanic fumes drifted up from cracks in the floor."
-            show_title=no
-        [/part]
-        [part]
-            story= _ "Distant rumbles and earthquakes made it difficult to keep steady footing. The very earth had come alive, heaving, struggling to be relieved of its centuries-old burden..."
-            show_title=no
-        [/part]
-    [/story]
-    {HTTT_TRACK {JOURNEY_17_NEW} }
-
-    #
-    # Map generator code
-    #
-
+    map_generation=lua
```

Comment:
(Note: This is more just a note of annoyance. Don't feel the need to revert the move. In fact, after viewing the full diff, I suspect it wouldn't make that much difference since a whole lot of other code was also moved from within `[settings]` and `[items]`.)